src/Axiom/UniquenessOfIdentityProofs.agda

```diff
@@ -62,13 +64,13 @@ module Decidable⇒UIP
 
   ≡-normalise : _≡_ {A = A} ⇒ _≡_
   ≡-normalise {a} {b} a≡b with a ≟ b
-  ... | yes p = p
-  ... | no ¬p = ⊥-elim (¬p a≡b)
+  ... | true  because  [p] = invert [p]
+  ... | false because [¬p] = ⊥-elim (invert [¬p] a≡b)
```

Comment:
I'm struggling to see the advantage of this change? Can you give an example of where the behaviour differs?

Reply:
Here I'm being cautious, and hoping that the principle of making case splits as late as possible might save us in some weird case. In particular, it's possible to imagine a difference in the `false` case, where we now have `(≡-normalise _ | false because <stuck>) = ⊥-elim <stuck>` rather than just being stuck.

Another rough principle was to pretend that old `Dec` never existed, and to rewrite stuff in the newly “obvious” way. This goes some way to explaining the `true` case, as well as wanting to keep consistency with the `false` case.

Reply:
Okay fair enough, thanks for the explanation.